internal/constants/files.go

```diff
@@ -2,3 +2,4 @@ package constants
 
 const Kubeconfig = "kubeconfig"
 const KubeconfigNoIngress = "kubeconfig-noingress"
+const AppsSubDomainName = "console-openshift-console"
```

Comment:
Please change the name to be more indicative of what this is related to (something like `DnsValidationAppsWildcardSubdomain`, separate if from the other constants (with newlines) because they're unrelated, and also add a comment explaining what it's about:

>Also, above that constant's definition please write a comment describing that this is an arbitrary subdomain of *.apps.<cluster>.<base> that we use in order to perform a DNS validation that checks the wildcard DNS has been applied by the user correctly.

Reply:
Not sure why the comment wasn't shown in the previous commit, uploading it again, also changed all the rest.